website/conferences/views.py

```diff
@@ -215,6 +215,14 @@ def conference_view(**kwargs):
         )
         projects = Node.find(query)
         submissions = projects.count()
+        downloads_list = [
```

Comment:
@chennan47 Indeed, the slowness would affect the main meeting page, which means it would affect more users than just the users for a single conference. When you say it takes a few seconds--do you mean a few seconds were *added* to the original load time? Anything that adds on the order of seconds is a very long time.